hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/FileSystemViewHandler.java

```diff
@@ -284,6 +284,13 @@ private void registerFileSlicesAPI() {
       writeValueAsString(ctx, dtos);
     }, true));
 
+    app.get(RemoteHoodieTableFileSystemView.ALL_EXCLUDE_FILEGROUPS_FOR_PARTITION_URL, new ViewHandler(ctx -> {
```

Comment:
ALL_EXCLUDE_FILEGROUPS_FOR_PARTITION_URL -> REPLACED_FILEGROUPS_FOR_PARTITION_URL

Lets use a single consistent name "replaced" instead of exclude everywhere.

Reply:
changed